lib/yard/cli/yardoc.rb

```diff
@@ -393,6 +393,7 @@ def print_list
       # @param [Array<String>] files the set of documentation files
       def add_extra_files(*files)
         files.map! {|f| f.include?("*") ? Dir.glob(f) : f }.flatten!
+        files.sort!
```

Comment:
This would change file order when files were listed explicitly, which is currently a supported feature. It's fairly common to want to manually sort extra files, for example, like putting your Getting Started Guide before more advanced documentation-- this is especially useful in the guide template, where document order is purposeful: https://github.com/lsegal/yard/blob/master/.yardopts_guide#L9

Sorting makes sense for a README, and even for file globbing, but not so much for the finalized list.